src/index.js

```diff
@@ -41,10 +42,11 @@ for (let i = 0; i < process.env.NDJSON_WORKERS; i++) {
 }
 
 const port = process.env.SERVER_PORT || 3000;
+const url = `mongodb://${process.env.DB_HOST}:${process.env.DB_PORT}`;
```

Comment:
Is there a way that we can make this url setting an intelligent default as part of the mongoUtil.Connection class? (i.e. if it's not otherwise set, set it to this). I'm worried that we're doing a bit of repeating setting this across the application.

Reply:
Not sure, but I will look into this!